pyaedt/circuit.py

```diff
@@ -798,14 +798,14 @@ def import_touchstone_solution(self, filename, solution_name="Imported_Data"):
 
     @pyaedt_function_handler()
     def export_touchstone(
-        self, solution_name=None, sweep_name=None, file_name=None, variations=None, variations_value=None
+        self, setup_name=None, sweep_name=None, file_name=None, variations=None, variations_value=None
```

Comment:
@fwlfHS maybe we can add the same parameters (renormalization and impedance) to the methods in circuit and in 3d layout. What do you think?